doc/autoscale/experiment/README.md

```diff
@@ -1,26 +1,40 @@
 # Auto-scaling Experiment Design
 
+## Purpose
 
-## Environment Enviroment
+To verify the effectiveness of PaddlePaddle's fault-tolerance and auto-scaling mechanism.
 
-- Kubernetes v1.6 cluster with 133 nodes.
-- PaddleCloud with latest develop branch installed.
-- We will train the [recognize_digits](https://github.com/PaddlePaddle/cloud/tree/develop/demo/recognize_digits) model in the experiment.
+## Metrics
 
+How the effectiveness is measured.
 
-## Experiment Metric
+1. Cluster computing resource overall utilization.
+    - the higher the better.
+    - higher utilization means less resource is idle.
+1. Task average pending time.
+    - the less the better.
+    - the less pending time the earlier developers and researchers can start seeing the training cost curve, and the better they can verify the training algorithm effectiveness.
+    - This is a common pain point of researchers with the internal cloud.
+1. Task average execution time.
+    - the less the better.
+    - average execution time is another way of measuring computing resource utilization. the less execution time, the higher overall utilization.
```

Comment:
Jobs may spend more time to run when total resource requested is more than what we have in the cluster because the scaler will scale down some low-priority jobs to ensure higher priority jobs. So when inspecting the results of the experiment, the less the average job running time increases, the better the scaler performances.